Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -176,6 +176,9 @@ def subnext(self):
 steps['EGMNano_mc13.0'] = merge([{'-s':'NANO:@EGM,DQM:@nanoAODDQM', '-n' : '1000'},
                                  steps['NANO_mc13.0']])
 
+steps['BTVNANO_data13.0']=merge([{'-s' : 'NANO:@PHYS+@BTV',
+                                    '-n' : '1000'},
```

Comment:
because `@BTV` always contains the `@PHYS`, then here (below), it should only be `NANO:@BTV`

Reply:
I try to remove it, but still not work with NANO:@BTV only

Reply:
that’s not supposed to fix anything, indeed. just configure things consistently